### PR TITLE
ISSUE#10: fix lms view for platform versions before hawthorn

### PR DESCRIPTION
**Pull request by msaqib52**
This PR fixes issue #10

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @msaqib52! I've created [OSPR-4378](https://openedx.atlassian.net/browse/OSPR-4378) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by natabene** [comment]
@msaqib52 Thank you for your contribution.

**Comment by natabene** [comment]
@davestgermain Can you give this a look when you have a chance?

File: edx_zoom/zoom_consumer.py [hunk 1]
```diff
@@ -84,21 +85,24 @@ def _get_context_for_template(self):
...
-        except Exception:
-            ctx['is_studio'] = True
         return ctx
 
     def student_view(self, context):
         fragment = Fragment()
         lti_loader = ResourceLoader('lti_consumer')
         loader = ResourceLoader(__name__)
-        context.update(self._get_context_for_template())
+        context = self._get_context_for_template()
```
Comment (davestgermain) [inline edx_zoom/zoom_consumer.py:94]: Is this change necessary? The context passed in will be lost. Is that intended?

Comment (msaqib52) [reply edx_zoom/zoom_consumer.py:94]: It would work fine in LMS without this change, but if someone is calling xblock View directly and passes some query params then context becomes QueryDict object and the `update` method adds zoom xblock context to a query dict which the template is unable to read because it renders as: `{'key': ['value']}`. Since we are not using any thing from the context passed so i simply skipped it.

**Comment by edx-webhook** [comment]
@msaqib52 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-zoom&entry.867055334=https://github.com/edx/edx-zoom/pull/11&entry.1484655318=https://github.com/msaqib52&entry.752974735=2020-04-12+17:19&entry.1917517419=2020-04-13+17:39&entry.2133058324=Yes) so we can improve your experience in the future.
